compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/writer/BIRBinaryWriter.java

```diff
@@ -54,7 +54,8 @@
  *
  * @since 0.980.0
  */
-public class BIRBinaryWriter {
+public class
+BIRBinaryWriter {
```

Comment:
why did we break this line?

Reply:
looks like this was by mistake 😬 will revert